File: DevcampPracticalProjectMocks/c#-project-mock/ProjectMock/ProjectMock/Controllers/PeersController.cs
using System.IO;
using Microsoft.AspNetCore.Hosting;
using Microsoft.AspNetCore.Mvc;
using Newtonsoft.Json;

namespace ProjectMock.Controllers
{
    [Route("[controller]")]
    public class PeersController : Controller
    {
        private IHostingEnvironment _hostingEnvironment;

        public PeersController(IHostingEnvironment hostingEnvironment)
        {
            this._hostingEnvironment = hostingEnvironment;
        }

        [HttpPost("connect")]
        public IActionResult Connect()
        {
            object jsonObject = GetJSON("peer-connected.json");

            return Json(jsonObject);
        }

        [HttpGet("")]
        public IActionResult List()
        {

            object jsonObject = GetJSON("peers.json");

            return Json(jsonObject);
        }

        [HttpPost("notify-new-block")]
        public IActionResult NotifyNewBlock()
        {

            object jsonObject = GetJSON("notify-new-block.json");

            return Json(jsonObject);
        }

        public object GetJSON(string name)
        {
            string path = Path.Combine(_hostingEnvironment.WebRootPath, $"mocks/{name}");

            string json = System.IO.File.ReadAllText(path);

            object jsonObject = JsonConvert.DeserializeObject(json);

            return jsonObject;
        }

    }
}

File: DevcampPracticalProjectMocks/java-project-mock/src/main/java/org/softuni/web/PeersController.java
package org.softuni.web;

import org.softuni.utils.ResponseEntityBuilder;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

@Controller
@RequestMapping("/peers")
public class PeersController {

    public PeersController(){}

    @RequestMapping
    public ResponseEntity<String> peers(){
        return ResponseEntityBuilder.buildJsonResponseEntity("peers.json");
    }

    @RequestMapping(value = "/connect", method = RequestMethod.POST)
    public ResponseEntity<String> connect(){
        return ResponseEntityBuilder.buildJsonResponseEntity("peer-connected.json");
    }

    @RequestMapping(value = "/notify-new-block", method = RequestMethod.POST)
    public ResponseEntity<String> notifyNewBlock(){
        return  ResponseEntityBuilder.buildJsonResponseEntity("notify-new-block.json");
    }

}
